src/ResourceManager/Insights/Commands.Insights/LogsCmdletBase.cs

```diff
@@ -165,10 +165,6 @@ protected string ProcessParameters()
         {
             string queryFilter = this.ProcessGeneralParameters();
             var result = this.ProcessParticularParameters(queryFilter);
-            this.WriteIdentifiedWarning(
```

Comment:
@pkamath13 @gucalder just to confirm, this breaking change was made in November, correct?

Reply:
Yes. The field EventChannels is not there in EventData anymore.